### DISCUSSION TITLE
Miniconda problems

### DISCUSSION THREAD

**Discussion by halieute** [giswqs/geog-312#97] [Q&A] [answered]
Goog morning Prof i have a big problem related to my miniconda which dosen't allow to to do nothing on my anaconda prompt. These are some screenshot.  I tried also to put the Script path on the system environment, it is still not working.
![Screenshot 2024-10-07 004616](https://github.com/user-attachments/assets/8f05f7be-9f46-4172-913e-29a6378249a4)
![Screenshot 2024-10-07 004635](https://github.com/user-attachments/assets/23deb527-ffad-4a1f-bb5a-bddbe86b7848)
![Screenshot 2024-10-07 004649](https://github.com/user-attachments/assets/0417f27b-0852-4bd1-ace5-090ebcba92ca)
![Screenshot 2024-10-07 004704](https://github.com/user-attachments/assets/5f3b82cb-37a8-4c92-9bc8-9c6c17ecfc44)
![Screenshot 2024-10-07 004717](https://github.com/user-attachments/assets/2d6ce1c6-af5d-493d-bb13-e0e437d3b3a8)
![Screenshot 2024-10-07 004727](https://github.com/user-attachments/assets/2969e4d8-cf0c-48d5-a952-154e9fdf8752)

**Comment by giswqs** [comment, marked as answer]
This is a conda problem specifc to your computer. If conda does not work for you, try uv
https://youtu.be/-YnfmHBHkWQ?si=bUJXNhFrMB3qcsMf
